config/src/config/network_config.rs

```diff
@@ -59,6 +59,8 @@ pub struct NetworkConfig {
     // in case some peers don't have well defined addresses.
     pub seed_pubkeys: SeedPublicKeys,
     pub max_frame_size: usize,
+    // Enables proxy protocol on incoming connections to get original source addresses
```

Comment:
can you point to the spec?

Reply:
The proxy protocol spec? https://www.haproxy.org/download/1.8/doc/proxy-protocol.txt